src/packs/checker.rs

```diff
@@ -339,3 +364,199 @@ fn get_checkers(
         }),
     ]
 }
+
+fn remove_reference_to_dependency(
+    configuration: &Configuration,
+    package_name: &str,
+    dependency_name: &str,
+) {
+    let existing_pack: &Pack =
+        configuration.pack_set.for_pack(&package_name).unwrap();
+    let file_name = format!(
+        "{}/{}/package.yml",
+        configuration.absolute_root.display(),
+        existing_pack.relative_path.display()
```

Comment:
Is there a better way to get the full path of a pack's package.yml?

Reply:
Yep – you can just do: `existing_pack.yml`!